yarn_api_client/resource_manager.py

```diff
@@ -317,7 +333,7 @@ def cluster_application_kill(self, application_id):
         :rtype: :py:class:`yarn_api_client.base.Response`
         """
 
-        data = '{"state": "KILLED"}'
+        data = {"state": "KILLED"}
```

Comment:
Wow - nice catch.

Reply:
In fact, it's not an error, and completely acceptable in our case.  However, working with strings is typically bringing problems unlike direct dictionaries since they're properly parsed and syntax highlighted.